src/pages/Login/index.js

```diff
@@ -9,12 +9,13 @@ import { useDispatch } from 'react-redux';
 import { api } from 'services/api';
 import { useLoginMutation } from 'services/auth/auth';
 import { setLoggedInUser } from 'utils/auth';
+import VideoContainer from './VideoContainer';
 import useAuth from 'hooks/useAuth';
 import useTranslation from 'hooks/useTranslation';
 import routesPaths from 'routes/routesPaths';
 import Input from 'components/form/Input';
 import Button from 'components/common/Button';
-
+import './styles.css';
 import '../../styles/form.css';
```

Comment:
If you're using absolute imports, this can be modified to be import 'styles/form.css'